### PR TITLE
Fix imports, refactor models

### PR DESCRIPTION
**Pull request by lsav**
- get rid of context files and put path insertions into __init__ files
- move models to its own folder
- put all the constants, tuples, and errors in one place for easy reuse
- clean up imports in general
- pedantic renaming of database tables to be more consistent

### CODE REVIEW

File: kaos/models/models.py [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+        Args:
+            platform_id: The unique ID of the satellite.
+            timestamp: The time in seconds since the Unix epoch.
+
+        Returns:
+            The segment with a matching platform_id, and where start_time <= timestamp <= end_time.
+            If more than one segment matches, which occurs only if the timestamp is the boundary
+            of two segments, return the later segment. If no segment matches, return None.
+        """
+        # pylint: disable=undefined-variable
```
Comment (j-asefa) [inline kaos/models/models.py]: Are we trying to remove all these comments?

File: kaos/models/models.py [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+        return (OrbitSegment.query.filter(OrbitSegment.platform_id == platform_id,
+                                          OrbitSegment.start_time <= timestamp,
+                                          OrbitSegment.end_time >= timestamp)
+                                  .order_by(OrbitSegment.start_time.desc())
+                                  .first())
+        # pylint: enable=undefined-variable
```
Comment (j-asefa) [inline kaos/models/models.py]: Here as well

File: kaos/__init__.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 """KAOS application initialization code."""
 
 from flask import Flask, jsonify
-from api.errors import APIError
```
Comment (ZeyadTamimi) [inline kaos/__init__.py]: Separate package imports from library imports

File: test/utils/test_utils.py [hunk 4]
```diff
@@ -2,7 +2,6 @@
 
 from ddt import ddt, unpack, data
 
-import context
 from kaos.utils.time_conversion import utc_to_unix
```
Comment (ZeyadTamimi) [inline test/utils/test_utils.py]: Double check this, WUT.

File: kaos/models/models.py (hunk 1 above)
Comment (lsav) [reply kaos/models/models.py]: Nope, just merging 2 sections that both had pylint-disables

**Pull request merged**